### DISCUSSION TITLE
Feature Suggestion: Hide items when grouping by a field

### DISCUSSION THREAD

**Discussion by stuartcarnie** [community/community#7317] [Projects and Issues]
Love the projects feature so far, great work!

Feature suggestion: 
As a user when creating a grouped view, I would like the ability to select which items from the group are visible, to enhance the ability for me to tailor it futher. This selection would be persisted with the view.

**Comment by CodeCox** [comment]
Add a filter to the grouped view for further customisation:
[See doc: Filter](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/customizing-your-project-views#filter)

(Apologies if you already knew this and meant a more advanced type of custom selection.)
